### ISSUE TITLE
Regsiter Tab - Input 'Name' character length issue.

### ISSUE THREAD

**Issue by santoshmondal** [AnthonyNahas/ngx-auth-firebaseui#294]
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?

macOS Sierra

### Versions

Angular CLI: 8.0.1
Node: 10.15.3
OS: darwin x64
Angular: 8.0.1

### Repro steps

Register name input length is hard coded. It does not accept more than 20 characters. Please look into this. 
[https://ngx-auth-firebaseui.firebaseapp.com/examples/appearance](https://ngx-auth-firebaseui.firebaseapp.com/examples/appearance)

### Desired functionality

Please remove the Character length hard code. A configurable option would be helpful.

**Comment by AnthonyNahas** [comment]
I will add this request to the next release

**Comment by santoshmondal** [comment]
Thank You.

**Comment by AnthonyNahas** [comment]
this is now available in v3.3.0 (updated by @jeandat) 👍
